src/lazy/binary/raw/v1_1/value.rs

```diff
@@ -282,6 +282,7 @@ impl<'top> LazyRawValue<'top, BinaryEncoding_1_1> for &'top LazyRawBinaryValue_1
 
 #[derive(Debug, Copy, Clone)]
 pub enum DelimitedContents<'top> {
+    // I believe this is intended to be a flat version of Option<DelimitedContents>
```

Comment:
Note to self? Or the start of a doc comment that you forgot to finish?

Reply:
Oops, this was a note to self. I wasn't sure if `None` meant "empty contents" or "not applicable". It's the latter. I'll remove the comment, thanks.